phlib/include/kphcomms.h

```diff
@@ -14,7 +14,7 @@
 
 #include <kphmsg.h>
 
-NTSTATUS KphFilterLoadUnload(
+NTSTATUS NTAPI KphFilterLoadUnload(
```

Comment:
These functions are not exported or external.

Reply:
3rd party projects which use the kphlib may link against un exported functions, and if they by default use a different calling convention then __stdcall a fix like this is required.